source/pool/contracts/Pool.sol

```diff
@@ -252,37 +246,43 @@ contract Pool is IPool, Ownable2Step {
     uint256 _minimumAmount
   ) internal returns (uint256, bytes32[] memory) {
     if (_claims.length <= 0) revert ClaimsNotProvided();
-    uint256 _totalScore = 0;
+    uint256 _totalValue = 0;
     bytes32[] memory _rootList = new bytes32[](_claims.length);
```

Comment:
This should be `treeList` as we are more interested in which trees were claimed than what their roots were.